src/Mod/Assembly/App/AssemblyObject.cpp

```diff
@@ -142,13 +143,14 @@ App::DocumentObjectExecReturn* AssemblyObject::execute()
     return ret;
 }
 
-int AssemblyObject::solve(bool enableRedo, bool updateJCS)
+int AssemblyObject::solve(bool enableRedo, bool updateJCS, bool bundleFixedJts)
```

Comment:
We've already got significant boolean blindness problems with this method: is there another approach that would make sense here? The classic solution is a set of enumerations.

Reply:
Essentialy this is bit flags, so I'd replace it with bit flags kind of enum. So `AssemblySolve::EnableRedo | AssemblySolve::BundleFixedJts` as replacement for `true, false, true`

Reply:
Actually this argument is not needed it's cleaner without it so I'll just remove it